File: src/main/java/problems/Uri1070_SeisNumerosImpares.java
package problems;

import java.io.IOException;
import java.util.Scanner;

public class Uri1070_SeisNumerosImpares {

    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);
        int i = 0;
        int n = ler.nextInt();

        while (i < 6) {
            if (n % 2 != 0) {
                i++;
                System.out.println(n);
            }
            n++;
        }
    }
}

File: src/main/kotlin/problems/Uri1070_SeisNumerosImpares.kt
package problems

import java.util.Scanner

fun main(args: Array<String>) {
    val ler = Scanner(System.`in`)
    var i = 0
    var numero = ler.nextInt()

    while (i < 6) {
        if (numero % 2 != 0) {
            i++
            println(numero)
        }
        numero++
    }
}

/*
URI Online Judge | 1070
Seis Números Ímpares
Adaptado por Neilor Tonin, URI  Brasil

Timelimit: 1
Leia um valor inteiro X. Em seguida apresente os 6 valores ímpares consecutivos a partir de X, um valor por linha, inclusive o X ser for o caso.

Entrada
A entrada será um valor inteiro positivo.

Saída
A saída será uma sequência de seis números ímpares.

Exemplo de Entrada	Exemplo de Saída
8

9
11
13
15
17
19
 */